spec/presenters/person_presenter_spec.rb

```diff
@@ -5,27 +5,38 @@
   let(:person) { profile_user.person }
 
   describe "#as_json" do
-    context "with no current_user" do
-      it "returns the user's public information if a user is not logged in" do
+    context "with no current_user (a user is not logged in)" do
```

Comment:
I I think the current description is clear, to the presenter current_user being present or not is what matters, it has no concept of that meaning "somebody is signed in"